### ISSUE TITLE
add a double-click event on the row

### ISSUE THREAD

**Issue by domatskiy** [matfish2/vue-tables-2#579]

**Comment by talapai** [comment]
You can use row-click event.
For example:
 ```Event.$on('vue-tables.row-click', this.toggleChildRow);```
``` 
toggleChildRow({event, row}) {
      if(event.type === 'dblclick') {
        this.doSomething()
      }      
    },```
See here: https://github.com/matfish2/vue-tables-2#events

**Comment by elanclarkson** [comment]
I have noticed that an event for the double click gets sent here in addition to the single clicks, so if you double click a row this function is triggered 3 times - once with event.type 'dblclick' and twice with event.type 'click'.

In my case this was making my toggle appear to not work because the 'double click' was toggling 3 times. I got around it by only letting single clicks through:

`Event.$on('vue-tables.row-click', data => {
			// Filter out double clicks (event.type === 'dblclick') because it gets sent at the same time as two 2 fast single clicks
			// I.e. two clicks sends 3 events and makes it feel like the toggle isn't working
			if (data.event.type === 'click') {
				this.toggle(data.row);
			}
		});`

**Comment by masterkain** [comment]
sorry to resume and old topic, but how effectively use double clicks?

```
EventBus.on("vue-tables.row-click", ({ event, row }) => {
  console.log(event.type);
  if (event.type === "dblclick") {
    playMedia(row);
  }
});
```

the `event.type` here is always `click`

**Comment by matfish2** [comment]
@masterkain fixed on version 0.4.3

**Comment by masterkain** [comment]
> @masterkain fixed on version 0.4.3

thanks @matfish2, the update is still not on git.xscode.com, once it is I'll report back 👍
